### ISSUE TITLE
where has middleware option gone? and why can't i use `db.js` anymore?  in latest v1-betta versions

### ISSUE THREAD

**Issue by bchenSyd** [typicode/json-server#1617]
I feel v0.xx is way better than v1-betta

how come all good features are gone???

**Comment by bchenSyd** [comment]
not imporessed with a big `db.json`
i have hundreds of routes, and I don't want a ginormous ,  monolithic  `db.json`  file. How do I spread json object to its own files?
thanks

**Comment by Strandedpirate** [comment]
I agree, it should watch a single file OR a folder full of files. Single file is great for onboarding, but you quickly out grow it in a matter of days. 

Project appears to be completely dead at this point. ..
